packages/protocol/contracts/governance/Validators.sol

```diff
@@ -918,7 +918,7 @@ contract Validators is
     uint256 balance = getLockedGold().getAccountTotalLockedGold(account);
     // Add a bit of "wiggle room" to accommodate the fact that vote activation can result in a 1
     // wei rounding error.
-    return balance.add(10) >= getAccountLockedGoldRequirement(account);
+    return balance.add(1) >= getAccountLockedGoldRequirement(account);
```

Comment:
@asaj any rationale for 10 as opposed to 1?
does it have to do with multiple vote activations?
this feels like an edge case that we don't account for elsewhere

Reply:
My guess would be to for sure overshoot whatever the rounding error can be? But yeah, would love @asaj's input before this gets merged.

Reply:
Yeah it was just for an additional margin of safety. I would argue for keeping it given I don't see much of a benefit of reducing it to 1, but I don't feel terribly strongly